cms/djangoapps/contentstore/utils.py

```diff
@@ -160,7 +160,10 @@ def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
 
 def course_image_url(course):
     """Returns the image url for the course."""
-    loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    try:
+        loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    except InvalidKeyError:
+        return None
```

Comment:
Now that you've changed this to return `None`, but haven't updated any other code, this seems suspect. What happens to the callers of this function, if in fact there is no course image? It looks like there are uses of this function directly in templates, which is going to give the html `<img src="None" ...`, which is pretty clearly wrong.